### DISCUSSION TITLE
GoogleCalendar.get_event() missing 1 required positional argument: 'event_id'

### DISCUSSION THREAD

**Discussion by DeveloperMaleek** [kuzmoyev/google-calendar-simple-api#137] [Q&A] [answered]
I am trying to get an event by id, but every time I do that, it says 
GoogleCalendar.get_event() missing 1 required positional argument: 'event_id'

here is my code
`get_calender = calendar.get_event(event_id='u43olol8qnk9rv43nb32tr8hn4')`
`print(get_calender)`

**Comment by kuzmoyev** [comment, marked as answer]
Can you please provide the whole traceback (error message) and the code where you initialize `calendar`?

**Reply by kuzmoyev** [reply]
Just an assumption: you might be calling the `get_event` on `GoogleCalendar` class instead of its instance. If so, you need to initialize it first like 
```
calendar = GoogleCalendar()
```
